src/docs/platform-architecture-reference/platform-automation.md

```diff
@@ -0,0 +1,82 @@
+- `Deployments`, `deploymentconfigs` and `statefulsets` that are crashing constantly 
+
+Emails are sent to the Product Owner and Technical Leads registered for the Product in the [Registry](https://registry.developer.gov.bc.ca/). If you want to change who gets the emails, go update the contacts in the Registry.
+
+## Deployments and deploymentconfigs that have not changed in a year 
+
+This tool checks the `lastUpdateTime` of the `Progressing` block in the `status` field of the Deployment or DeploymentConfig. If it is close to a year old, but not over, a warning email will be sent. If the deployment is managed by ArgoCD then a warning email is sent asking for action to be taken. This is because ArgoCD will just scale it back up after the tool scaled it down. If not managed by ArgoCD and the timestamp is over a year then the deployment is scaled to zero replicas and an email is sent.
+
+You can check the timestamp on your deployment in the YAML under `status`.
+ 
+ ```yaml
+   conditions:
+  - lastTransitionTime: "2021-11-25T17:04:01Z"
+    lastUpdateTime: "2021-11-25T17:04:11Z"
+    message: replication controller "backup-postgresql-1" successfully rolled out
+    reason: NewReplicationControllerAvailable
+    status: "True"
+    type: Progressing
+  ```
+
+## Deployments, deploymentconfigs and statefulsets that are crashing constantly
+
+This tool looks at the restart count of pods. If a pod has more than 100 restarts it looks for the controlling object of the pod, be it a Deployment, DeploymentConfig, or StatefulSet. If the object is managed by ArgoCD then a warning email is sent asking for action to be taken. This is because ArgoCD will just scale it back up after the tool scaled it down. If not managed by ArgoCD, the object is scaled to zero replicas and an email is sent.
+
+## Responding when deployments are scaled down by these tools
+
+Before you simply scale your pods back up, you should consider fixing the underlying issues that placed your application in one of these two groups in the first place.
+
+ If your application is crashing regularly, your technical team should investigate and fix the cause of these crashes. You can see the number of times your pod has restarted in the web console or via `oc get pods` to get an idea of if it is crashing a lot.
+ 
+ If your application has not changed in over a year, your technical team should build a new image, which ensures that your application is able to take advantage of any security patches from the past year. It is highly likely your image contains Critical security vulnerabilities if it has not been updated in over a year.
+ 
+ Maintaining your application is part of the [Memorandum of Understanding](https://digital.gov.bc.ca/cloud/services/private/onboard/#memorandum) that teams working on the platform agree to. More detailed docs on [maintaining an image](https://docs.developer.gov.bc.ca/maintain-an-application/#maintain-images) are also available.
```

Comment:
```suggestion
Keeping your application up-to-date is a requirement outlined in the [Memorandum of Understanding](https://digital.gov.bc.ca/cloud/services/private/onboard/#memorandum) agreed upon by teams operating on the platform. Additional comprehensive documentation on [maintaining an image](https://docs.developer.gov.bc.ca/maintain-an-application/#maintain-images) is also accessible for more detailed guidance.
```

Reply:
I think we need to be careful with wording here. The MOU includes a line for teams responsibility to 'Build, deploy and maintain your applications'. There is no specific reference to 'Keeping your application up-to-date'

Reply:
I thought about this one - you are right, the word "maintaining" is making reference to a continuous effort and it is used consistently throughout our documentation. You can leave it as such.

I used the word "keeping" as it is more of a word of everyday use, commonly known and to the point.